package.json

```diff
@@ -37,6 +37,7 @@
   "bundleDependencies": [],
   "dependencies": {
     "eslint-barrel-file-utils": "^0.0.10",
```

Comment:
need to update this `0.0.11` once https://github.com/thepassle/eslint-barrel-file-utils/actions/runs/9775456462 passes

Reply:
0.0.11 is published 👍

Reply:
Got it! I bumped `eslint-barrel-file-utils`'s version to v0.0.11 on the last commit.